main.go

```diff
@@ -67,6 +67,8 @@ func main() {
 
 	mux.HandleFunc("GET /blocks/pool/", handler.GetBlocksPool)
 
+	mux.HandleFunc("POST /transaction/add/", handler.AddTransactionToPool)
```

Comment:
У нас же по api такой эндпоинт ожидается

```suggestion
	mux.HandleFunc("POST /transactions", handler.AddTransactionToPool)
```